capture/colors.go

```diff
@@ -1,7 +1,9 @@
 package capture
```

Comment:
the only changes I made to this file are formatting and linting related, namely to shut my IDE up lmao.